Sources/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -767,6 +767,11 @@ private extension PurchasesOrchestrator {
                  completion: completion)
     }
 
+    func handleChangeStorefront() {
```

Comment:
This sounds weird. I'd rename it either `handleChangedStorefront` or `handleStorefrontChange`.

Reply:
Yep. Totally agree. I think `handleStorefrontChange` is more appropriated 👍🏼 Thanks

Reply:
Done ✅ [eec2fe8](https://github.com/RevenueCat/purchases-ios/pull/1583/commits/eec2fe86fac99cd5b5298f03f07a90a2916a7315)